tools/unix/generate_symbols.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 set -e -u
 
+if ! command -v optipng &> /dev/null
+then
+    echo -e "\033[1;31moptipng could not be found"
+    if [[ $OSTYPE == 'darwin'* ]]; then
+       echo 'run command'
+       echo 'brew install optipng'
+       echo 'to install it'
+       exit
+    fi
+    echo 'take a look to http://optipng.sourceforge.net/'
```

Comment:
Can it be installed for Windows? It would be great to support Windows developers in the nearest future.

Reply:
Yes, according to http://optipng.sourceforge.net/ it has win32 version.